src/lib/icons/IconErrorOutline.svelte

```diff
@@ -0,0 +1,36 @@
+<!-- source: DFINITY foundation -->
+<script lang="ts">
+  import { DEFAULT_ICON_SIZE } from "$lib/constants/constants";
+
+  export let size = `${DEFAULT_ICON_SIZE}px`;
+</script>
+
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width={size}
+  height={size}
+  viewBox="0 0 20 20"
+  fill="none"
+>
+  <path
+    stroke="currentColor"
```

Comment:
Does it work if you set the `stroke=currentColor` on the `svg` element?

i.e. space two attributes

Reply:
I've moved all the repeated attributes to the svg tag.